parambokeh/comms.py

```diff
@@ -225,6 +225,10 @@ def _handle_msg(self, msg):
                 with StandardOutput() as stdout:
                     self._on_msg(msg)
         except Exception as e:
+            # TODO: isn't this cutting out info needed to understand what's gone wrong?
+            # Since it's only going to the js console, maybe we could just show everything
+            # (error = traceback.format_exc() or something like that)? Separately we do need a mechanism
+            # to report reasonable messages to users, though.
```

Comment:
All true, if we can find a way to make tracebacks readable on the JS console I'd be very happy to change this (and port the change back to holoviews).